code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/file/property/SortableProperties.xtend

```diff
@@ -0,0 +1,58 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base.file.property
+
+import java.util.Properties
+import java.util.Set
+import java.util.Map
+import java.util.Comparator
+import java.util.Collections
+import java.util.TreeSet
+
+/**
+ * An implementation of {@link java.util.Properties} that allows for sorting its keys. Sorting is
+ * enabled by default. Note that this class is implemented in Java rather than Kotlin, because
+ * trying to override {@link java.util.Properties#entrySet()} in Kotlin results in a JVM signature
+ * platform declaration clash.
+ *
```

Comment:
Add a comment that this class is a copy of the SortableProperties class from the Java Base Generator and add a TODO that we need to factor both classes out into a common JAR in the future.